### PR TITLE
make ruby proxy worker accept script location

### PR DESCRIPTION
**Pull request by pongad**
This is done so that we can test other PHP clients.

### CODE REVIEW

File: src/ruby/qps/proxy-worker.rb [hunk 1]
```diff
@@ -155,6 +157,10 @@ def proxymain
     opts.on("-c", "--[no-]use_protobuf_c_extension", "Use protobuf C-extention") do |c|
       options[:c_ext] = c
     end
+    opts.on("-b" "--php_client_bin [FILE]",
+        "PHP client to execute; path relative to this script") do |c|
```
Comment (apolcyn) [inline src/ruby/qps/proxy-worker.rb]: nit: indentation of this line looks two spaces too far

File: src/ruby/qps/proxy-worker.rb [hunk 2]
```diff
@@ -155,6 +157,10 @@ def proxymain
     opts.on("-c", "--[no-]use_protobuf_c_extension", "Use protobuf C-extention") do |c|
       options[:c_ext] = c
```
Comment (apolcyn) [inline src/ruby/qps/proxy-worker.rb]: small nit that is outside of the scope of this PR: we might want to change  `c_ext` key from symbol to string to be consistent with the other keys to the `options` hash.

File: src/ruby/qps/proxy-worker.rb (hunk 1 above)
Comment (pongad) [reply src/ruby/qps/proxy-worker.rb]: @apolcyn I'm not terribly proficient with Ruby. This line is a continuation of the line above though, so shouldn't it be doubly indented?

File: src/ruby/qps/proxy-worker.rb [hunk 3]
```diff
@@ -155,6 +157,10 @@ def proxymain
...
     end
+    opts.on("-b" "--php_client_bin [FILE]",
```
Comment (apolcyn) [inline src/ruby/qps/proxy-worker.rb]: should there be a comma after "-b"?

File: src/ruby/qps/proxy-worker.rb (hunk 1 above)
Comment (apolcyn) [reply src/ruby/qps/proxy-worker.rb]: In most other source files that are style checked, I believe such a continuation has two spaces, but I believe the style checker isn't running on this so I'm not actually certain

Comment (pongad) [reply src/ruby/qps/proxy-worker.rb]: I see. Done.

File: src/ruby/qps/proxy-worker.rb (hunk 3 above)
Comment (pongad) [reply src/ruby/qps/proxy-worker.rb]: Done. For my own information, is this valid Ruby...? I was still able to run tests despite this problem.

**Comment by pongad** [comment]
@apolcyn PTAL

**Comment by apolcyn** [comment]
still LGTM

**Comment by pongad** [comment]
@apolcyn If this looks good to you, could you merge? I do not have write permission in this repo.
